kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/NetworkAdminServiceImpl.java

```diff
@@ -1539,7 +1575,7 @@ private void wifiModeWait(String ifaceName, WifiMode mode, int tout) {
     // Private Methods
     //
     // ----------------------------------------------------------------
-    private void enableWifiInterface(String ifaceName) throws KuraException {
+    private void enableWifiInterface(String ifaceName, NetInterfaceStatus status, WifiMode wifiMode, WifiConfig wifiConfig) throws KuraException {
```

Comment:
@ibinshtok Could this signature be simplified? Maybe it's possible to use the wifiMode inside the wifiConfig. In this way we could promote, a bit, the readability.